webpack_config/base.config.js

```diff
@@ -29,6 +31,8 @@ const commonConfig = {
   }
 }
 
+console.log(`MODE: ${commonConfig.mode}`)
```

Comment:
do we want to kill this in prod?

Reply:
Though I guess this is just in webpack so nvm

Reply:
Oh, sorry, forgot to delete that. Thanks for catching.

Reply:
Also, the readme is updated with local env info from last PR. No major changes to pipeline in this branch.